docs/containers.conf.5.md

```diff
@@ -256,6 +256,9 @@ conmon_path=[
 ]
 ```
 
+**container_engine_env**=[]
```

Comment:
Should be just called `env`, but this should be in the "engine" section.
When used inside of the code it would be 
Engine.Env rather then Containers.Env